config/default.json

```diff
@@ -11,6 +12,8 @@
 		"Password": "none"
 	},
 	"Logging": {
-		"LogLevel": "none"
+		"LogLevel": "none",
+		"SlackErrorChannel": "none",
```

Comment:
Is there a reason this file uses `"none"` as a string instead of `null` or something falsier?

Reply:
I did it so that when I called the variable later, if l logged the variable and got 'none' I knew the variable had been created but wasn't updated with the proper value for the environment. Not very clean/efficient.   I fixed this now. They are all set to `null` and a script runs through them when the server loads and prints their values so I can check them up front. (https://github.com/lbryio/spee.ch/pull/170/commits/250ead165bd51c4b9812d514c724d320f526d90c)